Doc/library/asyncio-task.rst

```diff
@@ -223,6 +246,11 @@ Creating Tasks
    :exc:`RuntimeError` is raised if there is no running loop in
    current thread.
 
+   .. note::
+
+      :meth:`asyncio.TaskGroup.create_task` is a newer alternative
+      that allows for convenient waiting for a group of related tasks.
+
```

Comment:
Same here. The topic of creating tasks could present `TG.create_task` here first, and the raw function later as a less recommended *unstructured* alternative.